shell/platform/android/io/flutter/view/FlutterMain.java

```diff
@@ -54,6 +55,8 @@
         FlutterMain.class.getName() + '.' + FLX_KEY;
     public static final String PUBLIC_SNAPSHOT_BLOB_KEY =
         FlutterMain.class.getName() + '.' + SNAPSHOT_BLOB_KEY;
+    public static final String PUBLIC_FLUTTER_ASSETS_DIR_KEY =
```

Comment:
It looks like this was never wired up in `initConfig()`, so even if apps specify this key in their manifest, we're still looking in the default assets dir.

Reply:
Ah yes, thanks!